scripts/west_commands/runners/mdb.py

```diff
@@ -61,6 +61,12 @@ def mdb_do_run(mdb_runner, command):
     mdb_basic_options = ['-nooptions', '-nogoifmain',
                         '-toggle=include_local_symbols=1']
 
+    # make the build folder the place where MDB is run. Otherwise mdb will store the
+    # .sc.project folder in the place where twister is launched (so .sc.project folder will be
+    # shared across the runs)
+    if path.exists(mdb_runner.build_dir):
+        os.chdir(mdb_runner.build_dir)
```

Comment:
This is a little dirty IMO. Can you please add a `**kwargs` argument to `ZephyrBinaryRunner.popen_ignore_int` instead, so you can pass a `cwd` argument to the `Popen` constructor? That is, handle this in the exact same way we have previously extended `ZephyrBinaryRunner.check_output`, `check_call`, etc.

Reply:
Indeed, that is more elegant solution. Thanks!